components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/assets/js/consent_template_1.js

```diff
@@ -0,0 +1,4 @@
+var tempStr1 =
```

Comment:
let's have a meaningful variable name

Reply:
changed to `customConsentTempalte1` and `customConsentTempalte2` respectively